app/models/experiment.rb

```diff
@@ -306,7 +306,7 @@ def create_result_csv_for(moe_name)
     CSV.generate do |csv|
       csv << self.parameters.flatten + [moe_name]
 
-      simulation_runs.where({is_done: true, is_error: {'$exists' => false}}, {fields: %w(values result)}).each do |simulation_run|
+      simulation_runs.where({is_done: true, is_error: {'$exists' => false}}).each do |simulation_run|
```

Comment:
Filtrowanie pól stało się niepożądanym zjawiskiem? Chyba używamy tego w naszej inżynierce.